manticore/platforms/linux.py

```diff
@@ -2623,6 +2629,8 @@ def solve_to_fd(data, fd):
 
         ret = {
             'syscalls': repr(self.syscall_trace),
+            'argv': repr(self.argv),
```

Comment:
I _think_ you need to solve an example solution out of argv/envp here. Otherwise you'll get the repr for an Expression. You need to apply something like solve_to_fd() for each variable/argument.

Reply:
Thanks, will look into that